### ISSUE TITLE
Use brackets to define new entity in `Note` and `Partnership` blocks instead of `@`

### ISSUE THREAD

**Issue by liminal-dog** [DiamondDAO/chainverse-frontend#95]
i.e. `[[Diamond DAO]]` not `@DiamondDAO`

**Comment by jose-zamora** [comment]
@liminal-dog you mean `[[diamon DAO]]` instead of `#DiamondDAO` bc as you can see in the image bellow for creating tag its using `#`
<img width="557" alt="image" src="https://user-images.githubusercontent.com/108299975/177431776-e64c1414-b0cd-4a1a-b3b9-10cdc67fcdf5.png">

**Comment by liminal-dog** [comment]
Good question. So `tags` should continue to use the `#` syntax, i.e. `#knowledge-graphs`.

Entities should use the double bracket syntax instead of the `@` syntax.

So, for instance...

```
[[Diamond DAO]] is building the most comprehensive #knowledge-graph of web3 communities.
```

**Comment by clemp** [comment]
This is fixed in #109
